### PR TITLE
Add possibility to create multiple ebs

### PR DESCRIPTION
**Pull request by next0262**
I think this terraform module doesn't work for creating multiple ebs.  
I made some changes to enable allocating multiple ebs to runner. 
I guess my changes can resolve the issue #1769 as well.

### CODE REVIEW

**Comment by npalm** [comment]
@next0262 thanks, did you test the change?

**Comment by toast-gear** [comment]
https://github.com/philips-labs/terraform-aws-github-runner/pull/1807 please don't break the ability to provide `null` as part of this PR

**Comment by npalm** [comment]
@toast-gear thanks for the note. Maybe I overlook something. But providing the default `[]` empty array results in the same behavior as `null` in PR #1807

**Comment by next0262** [comment]
@toast-gear Thanks for your correction. I fixed it for keeping the ability for null.
@npalm yes, I did some tests with github and aws and i checked there is no problem.

**Comment by toast-gear** [comment]
I haven't tried your build I just wanted to highlight not to cause a regression with this change, if an empty list doesn't then crack on with removing null!

**Comment by next0262** [comment]
@toast-gear I understand. Thank you for your advice. Please tell me if my additional change has another problem.

**Comment by npalm** [comment]
cc @Hunqer woould you have time for a quick chck of this PR, to be sure it not conflicts your work

**Comment by next0262** [comment]
@Hunqer I am so looking forward to your check. I would really appreciate it .

**Comment by Hunqer** [comment]
@npalm  @next0262 Will check that today, and come back to you.

**Review by Hunqer — APPROVED** [review]
From my test looks like it not broking our workflow. @next0262 @npalm

**Comment by next0262** [comment]
@npalm I ask you to approve my changes if there is no problem

**Comment by npalm** [comment]
Will review and check the PR asap

**Comment by next0262** [comment]
@npalm Your check is done?

File: modules/runners/variables.tf [hunk 1]
```diff
@@ -47,8 +47,8 @@ variable "s3_location_runner_binaries" {
 
 variable "block_device_mappings" {
   description = "The EC2 instance block device configuration. Takes the following keys: `device_name`, `delete_on_termination`, `volume_type`, `volume_size`, `encrypted`, `iops`"
```
Comment (ScottGuymer) [inline modules/runners/variables.tf:49]: Can you add a type constraint for the mentioned keys instead of `map(string)`.

File: variables.tf [hunk 2]
```diff
@@ -225,8 +225,8 @@ variable "runner_allow_prerelease_binaries" {
 
 variable "block_device_mappings" {
   description = "The EC2 instance block device configuration. Takes the following keys: `device_name`, `delete_on_termination`, `volume_type`, `volume_size`, `encrypted`, `iops`"
-  type        = map(string)
-  default     = {}
+  type        = list(map(string))
```
Comment (ScottGuymer) [inline variables.tf:228]: Same comment as above.

File: modules/runners/variables.tf (hunk 1 above)
Comment (next0262) [reply modules/runners/variables.tf:49]: @ScottGuymer  I added type constraints on block_device_mappings

File: variables.tf (hunk 2 above)
Comment (next0262) [reply variables.tf]: Same reply as above.

**Comment by next0262** [comment]
@ScottGuymer @npalm I modified block_device_mappings of ubuntu example based on type constraints

**Review by npalm — APPROVED** [review]

**Pull request merged**